content/id/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -0,0 +1,186 @@
+---
+reviewers:
+- freehan
+title: EndpointSlice
+feature:
+  title: EndpointSlice
+  description: >
+    Pelacakan _endpoint_ jaringan yang _scalable_ pada klaster Kubernetes.
+
+content_template: templates/concept
+weight: 15
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+
+EndpointSlice menyediakan sebuah cara yang mudah untuk melacak _endpoint_ jaringan dalam sebuah
+klaster Kubernetes. EndpointSlice memberikan alternatif yang lebih _scalable_ dan lebih dapat diperluas daripada Endpoint.
```

Comment:
```suggestion
klaster Kubernetes. EndpointSlice memberikan alternatif yang lebih _scalable_ dan lebih dapat diperluas daripada Endpoints.
```
The object that EndpointSlice is replacing is called “Endpoints” (it's a plural name in the API).

Reply:
Noted. Thanks for pointing that out @sftim.

Reply:
Solved in https://github.com/kubernetes/website/pull/20166/commits/2e77f8a8acdbc6cb0cb503c32270c1b2871ae0e0